src/ImageSharp/Common/ParallelUtils/ParallelHelper.cs

```diff
@@ -45,7 +45,8 @@ public static void IterateRows(Rectangle rectangle, Configuration configuration,
             in ParallelExecutionSettings parallelSettings,
             Action<RowInterval> body)
         {
-            DebugGuard.MustBeGreaterThan(rectangle.Width, 0, nameof(rectangle));
+            Guard.MustBeGreaterThan(rectangle.Width, 0, nameof(rectangle));
```

Comment:
Maybe something like `$"{nameof(rectangle)}.{nameof(rectangle.Width)}"` or `nameof(rectangle.Width)` instead to make it clear which value is incorrect?

Reply:
Done